### PR TITLE
Remove Godeps/_workspace on `make clean`

### PR DESCRIPTION
**Pull request by thockin**
I thought this was in the main PR.  My bad.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -110,6 +110,7 @@ test_e2e_node:
 clean:
 	build/make-clean.sh
 	rm -rf $(OUT_DIR)
+	rm -rf Godeps/_workspace # Just until we are sure it is gone
```
Comment (eparis) [inline Makefile]: if it doesn't exist, does rm return error and the make fail?

Comment (eparis) [inline Makefile]: deleting line note, `rm -rf` returns true no matter what.

**Comment by eparis** [comment]
I switched to `release-note-none`    I think just the 'vendor/' PR should be enough of a release note about this stuff, no?

**Comment by thockin** [comment]
Sorry, I meant release-not-none, but hit the wrong one, apparently.

**Pull request merged**